src/pro/javacard/gp/AccessControl.java

```diff
@@ -0,0 +1,769 @@
+package pro.javacard.gp;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.bouncycastle.util.Arrays;
+
+import apdu4j.HexUtils;
+
+/**
+ * Access control Rules implementation (reference document : Secure Element Access Control Version 1.0).
+ *
+ * @author Bertrand Martel
+ */
+public class AccessControl {
```

Comment:
Maybe add SE prefix to class, to distinguish from arbitrary "access control". Or maybe a sub-package?